celt/arm/armcpu.c

```diff
@@ -191,7 +191,7 @@ static opus_uint32 opus_cpu_capabilities(void)
   return flags;
 }
 
-#elif defined(__FreeBSD__)
+#elif defined(HAVE_ELF_AUX_INFO)
```

Comment:
I'm not sure I understand why you need to change the __FreeBSD__ case to add OpenBSD. Not against your OpenBSD patch but would like to make sure we don't break FreeBSD. More specifically, your HAVE_ELF_AUX_INFO is added to only two out of the four build systems we support.

Reply:
Not every release of FreeBSD nor OpenBSD has the elf_aux_info() API. This is how I've done things with pretty much every other project I've added OpenBSD or FreeBSD/OpenBSD support. I have now added and fixed the CMake support.